dspace-api/src/test/java/org/dspace/app/bulkedit/MetadataImportIT.java

```diff
@@ -152,6 +152,22 @@ public void relationshipMetadataImporAlreadyExistingItemTest() throws Exception
 
     }
 
+    @Test
+    public void personMetadataImportTest() throws Exception {
+
+        String[] csv = {"id,collection,dc.title,person.birthDate",
+            "+," + collection.getHandle() + ",\"Test Import 2\"," + "2000"};
+        performImportScript(csv);
+        Item importedItem = findItemByName("Test Import 2");
+        assertTrue(
+            StringUtils.equals(
+                itemService.getMetadata(importedItem, "person", "birthDate", null, Item.ANY)
+                           .get(0).getValue(), "2000"));
+        context.turnOffAuthorisationSystem();
+        itemService.delete(context, itemService.find(context, importedItem.getID()));
```

Comment:
Tiny thing I just realized.  You likely can simplify this line to be `itemService.delete(context, importedItem);`  I don't think you should need the `itemService.find()` here...unless I'm missing something.

In any case, this is extremely minor. The code overall is good.